test/exec/continuous-integration.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+npm install
+
+# Setup the virtual frame buffer and export the DISPLAY env variable
+Xvfb :1 -screen 5 1024x768x8 &
```

Comment:
Please refer to our machines generically in public places. (I've edited your comment :-)